### DISCUSSION TITLE
有没有登录评论需验证的api或变量？

### DISCUSSION THREAD

**Discussion by w0oo** [walinejs/waline#1222] [Q&A] [answered]
例子：（注册后需验证邮箱，验证后可登录那种）

类似：https://lty-waline.vercel.app/ui
（不是我的）
那种
![Screenshot_2022-07-23-10-24-38-44_470b4cb16301e0ed7c107f89a0d69573](https://user-images.githubusercontent.com/105968998/180587105-ef96c49b-9785-49b0-a377-37bca83a9e00.jpg)

**Comment by lizheming** [comment, marked as answer]
你配置了邮箱相关的服务配置后，会自动有这个功能，注册确认邮件。
